akka-samples/akka-docs-java-lambda/src/test/java/docs/stream/StreamTcpDocTest.java

```diff
@@ -49,27 +50,25 @@ public static void tearDown() {
   }
 
   final FlowMaterializer mat = ActorFlowMaterializer.create(system);
-  
-  final InetSocketAddress localhost = new InetSocketAddress("127.0.0.1", 8889);
-  
+
   private final ConcurrentLinkedQueue<String> input = new ConcurrentLinkedQueue<String>();
   {
     input.add("Hello world");
     input.add("What a lovely day");
   }
-  
+
   private String readLine(String prompt) {
     String s = input.poll();
     return (s == null ? "q": s);
   }
-  
+
   @Test
   public void demonstrateSimpleServerConnection() {
     //#echo-server-simple-bind
-    final InetSocketAddress localhost = 
-        new InetSocketAddress("127.0.0.1", 8889);
+    final InetSocketAddress localhost =
+      SocketUtils.temporaryServerAddress();
```

Comment:
Yes. There is the same problem in scala docs as well. I'll look into it how to mask this and also how to reuse the code from akka tests for temporary server address.